AdminWebsite/AdminWebsite/Mappers/ParticipantResponseMapper.cs

```diff
@@ -39,6 +39,7 @@ public static List<ParticipantResponse> Map(this List<V2.ParticipantResponseV2>
                 Id = p.Id,
                 DisplayName = p.DisplayName,
                 HearingRoleCode = p.HearingRoleCode,
+                HearingRoleName = p.HearingRoleName,
```

Comment:
As with the bookings api PR, please can you add a test to verify that this property is mapped?